handlers.go

```diff
@@ -38,13 +40,16 @@ func (sf *Sunfish) AddFile(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 		w.WriteHeader(422) // unprocessable entity
 		if err := json.NewEncoder(w).Encode(err); err != nil {
-			panic(err)
+			sf.logger.Println("ERROR: Could not encode json response.")
+			return
 		}
+		sf.logger.Println("INFO: Could not process the uploaded Siafile.")
```

Comment:
Luke took the same approach and ended up with a log that grew like 20mb per day. I like to keep things lightweight, and don't really subscribe to the 'infinite data' mentality. Lots of data means lots of things to go through, especially since the valuable stuff is usually only a very small chunk of it anyway.

I'd rather keep the logs small and aim at the stuff we care about.

If you want to keep track of how people are making mistakes when doing uploads, instead of making a log entry have a tally that counts how many time each error is returned, and maybe save the upload attempt or something every nth error, where n is an exponentially growing number. (ie, save attempt 1, 2, 4, 8, etc.)

But I also think this might be a case of premature optimization. You want the logs to help you figure out stuff you can't figure out without them. I'm not sure we're at that stage.